### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by VGrondin** [xiaohu2015/SwinT_detectron2#16]
Hi, I am using SwinT with mask R-CNN and everything compile and was easy to setup.
However, when I use the pretrained weights (mask_rcnn_swint_T_coco17.pth) to finetuned the model on my custom dataset, I can't seem to acheive over 35  APbb. With ResNet I usually acheive over 60 APbb. Is the pretrained weight the one you got on coco or imagenet? Any tips on what could cause the AP difference? 

Thank you

**Comment by xiaohu2015** [comment]
@VGrondin Yes, I trained on coco17 datasets with pretrained imagenet weights. You can check the performance on the coco17 val with the weights.
